src/services/types.ts

```diff
@@ -608,6 +608,11 @@ namespace ts {
         triggerSpan: TextSpan;
     }
 
+    export interface IsValidIdentifierName {
+        isValid: boolean;

```

Comment:
Would it make more sense to just return `{ localizedErrorMessage: string | undefined }`? You would never return `{ isValid: true, localizedErrorMessage: "Nvm, here's an error message anyway" }`, right?

Reply:
Makes sense. Fixed